CONTRIBUTING.md

```diff
@@ -1,17 +1,65 @@
-Contributing
-============
+# Contributing
 
-TODO: Fill out your contributing process, or just stick with the default.
-
-Steps
------
+## Steps
 
 Thanks for your interest in sensu-go cookbook! If you'd like to help out please:
 
 1. Fork it
 1. Create your feature branch (`git checkout -b my-new-feature`)
-1. Add tests for the new feature; ensure they pass (`rake`)
+1. Add tests for the new feature; ensure they pass See [TESTING.md](./TESTING.md)
 1. Add a changelog entry under `## [Unreleased]` and the appropriate sub header for the type of change per our [guidelines](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)
 1. Commit your changes (`git commit -am 'Add some feature'`)
 1. Push to the branch (`git push origin my-new-feature`)
 1. Create a new Pull Request
+
+## How to Become a Maintainer
+
+### Types of Maintainers
+
+#### SME (Subject Matter Expert)
+
+##### Pre Conditions
+
+To become an SME maintainer you must meet one of the following conditions:
+
+1. Has made significant contributions to a particular repo
+2. Has specific area knowledge related to the content.
+
+The following is required:
+
+1. Have MFA (Multi Factor Authentication) enabled on your Github account
+
+##### Process
+
+1. Open an [issue](https://github.com/sensu/sensu-go-chef/issues/new) indicating that you would like to become a maintainer and your subject matter expertise.
+
+#### Maintainer
+
+Most maintainers are targetted at maintainting a specific set of repositiories based on language, function (plugin vs extension), Documentation, etc
+
+##### Pre Conditions
+
+1. Shown consistent and quality contributions to multiple repositories in the Sensu ecosystem.
+1. MFA enabled on your Github account
+1. Have setup PGP with github so you can sign commits to guarantee that your commits are your own
+
+##### Process
+
+1. Open an [issue](https://github.com/sensu/sensu-go-chef/issues/new) indicating that you would like to become a maintainer
+1. Endorsed by two or more active maintainers with no vetos against them
+1. Have acknowledged that they have read and will abide by the terms outlined in the [Code of Conduct](https://sensu.io/conduct)
+1. Will be invited to the organization, teams, slack channels, etc as outlined via internal maintainer processes.
+
+#### Org Maintainer
+
+An org maintainer is not scoped to specific projects (even if they mostly focus on an area), they provide OSS mentorship to other maintainers, manage permissions + credentials, set organization wide policies, etc.
```

Comment:
This can probably be axed and makes more sense in the context of the plugins organization rather than the single repository for chef.